section2/crud.rb

```diff
@@ -2,22 +2,26 @@ module Crud
   require 'bcrypt'
   puts "Module CRUD activated"
 
-  def self.create_hash_digest(password)
+  # def self.create_hash_digest(password)
+  def create_hash_digest(password)
     BCrypt::Password.create(password)
   end
```

Comment:
```suggestion
  def create_hash_digest
    BCrypt::Password.create(self.password)
  end
```

Reply:
For this, does it means only Class that have instantiated with `password` attribute only are allowed to use this method?

Reply:
self.password is an instance variable. read up about it